src/app/core/services/config/config.service.ts

```diff
@@ -304,6 +304,7 @@ export class ConfigService {
       this.subjectConfig
         .init(user)
         .then(() => this.analytics.setUserProperties(user))
+        .then(() => this.analytics.setUserProperties(user.attributes))
```

Comment:
will the other user properties (like userId, projectId, etc) still be set as usual?
also can we add some prefix to these properties so it is easy to see they are from attributes like `attribute_participant_group`

Reply:
Yup the others will be set as usual.